giskard/llm/client/base.py

```diff
@@ -16,6 +16,7 @@ class LLMFunctionCall:
 class LLMOutput:
     message: Optional[str] = None
     function_call: Optional[LLMFunctionCall] = None
+    tool_calls: Sequence[LLMFunctionCall] = field(default_factory=list)
```

Comment:
For this moment it is ok for a backward compatibility, but I believe, that by the end of the day we need to keep only "tool" and not "function" naming convention and thus related variables like "function_call"

Reply:
Yes I'm not sure if `function_call` are being used in other PR's and might be used by some users having developed there own test.

I guess it's okay to keep `LLMFunctionCall` for now since only tool supported for now is of type `function`. I guess we will need to update it once we have .